core/exception/interrupt_spec.rb

```diff
@@ -0,0 +1,35 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+describe "Interrupt.new" do
```

Comment:
I think it would be great to add this spec example:
```ruby
  it "is a subclass of SignalException" do
    Interrupt.superclass.should == SignalException
  end
```